controller/controller.go

```diff
@@ -224,6 +243,7 @@ func (c *Controller) Reconcile(req reconcile.Request) (reconcile.Result, error)
 	res, err := c.reconcile(ctx, req)
 	if err != nil {
 		errorGauge.Inc()
+		c.sentry.Capture(ctx, err)
```

Comment:
Does `Capture` return an error that we should handle?

Reply:
No. That function is a wrapper around the client library `CaptureException` func that returns an EventID pointer or nil

Reply:
Linter should catch that if it is the case.